lib/v_3.0.0/init.js

```diff
@@ -99,7 +99,7 @@ module.exports = function(appName, options) {
     console.log(chalk.bgCyan('////////////////////////////////////////////////////////'));
     console.log();
     shell.cd("server");
-    shell.sed('-i', `yote`, appName, `./config.js`);
+    shell.sed('-i', `yote`, appName, [`./config/default.js`, `./config/development.js`, `./config/staging.js`, `./config/production.js`]);
```

Comment:
+1. i never got around to changing these so I guess we'll have to live with it!

Reply:
lmao